api/src/opentrons/protocol_engine/commands/command.py

```diff
@@ -76,6 +76,13 @@ class BaseCommandCreate(GenericModel, Generic[CommandParamsT]):
             " and module setup, like pre-heating."
         ),
     )
+    key: Optional[str] = Field(
+        None,
+        description=(
+            "Command key generated by UI and consumed by the api. By adding this field we are enabeling "
+            " protocol analysis for JSON v6."
+        ),
```

Comment:
Remember that the context of these description is that of a HTTP API user, not of an Opentrons developer.

```suggestion
        description=(
            "A key value, unique in this run, that can be used to track"
            " the same logical command across multiple runs of the same protocol."
            " If a value is not provided, one will be generated."
        ),
```